src/com/spikes2212/utils/Monitor.java

```diff
@@ -0,0 +1,34 @@
+package com.spikes2212.utils;
+
+import java.util.function.Supplier;
+
+import edu.wpi.first.wpilibj.PowerDistributionPanel;
+
+public class Monitor {
+	private static PowerDistributionPanel pdp;
+	private static double high_voltage = 10;
+	private static double low_voltage = 7;
+
+	public Monitor() {
+		pdp = new PowerDistributionPanel();
+	}
+
+	public static void setHigh_voltage(double high_voltage) {
+		Monitor.high_voltage = high_voltage;
```

Comment:
You should make sure the high voltage is higher than the low voltage, and vice versa